src/app/components/feed/jh-feed.directive.js

```diff
@@ -61,6 +61,10 @@
             // Reset the warning timeout
             mutedWarningTimeout = now();
           });
+          scope.$on('muted.Join', function() {
+            mutedWarningTimeout = secondsFromNow(60);
```

Comment:
Wouldn't it be better to set it to `now()` or to `secondsFromNow(3)` ? It would have a similar effect but with more consistent times (the same time than muting myself or the same one that having been muted). Isn't it?